src/mrpro/data/KHeader.py

```diff
@@ -171,13 +171,13 @@ def from_ismrmrd(
             parameters['n_coils'] = header.acquisitionSystemInformation.receiverChannels
 
         if header.sequenceParameters is not None:
-            if header.sequenceParameters.TR:
+            if any(header.sequenceParameters.TR):
```

Comment:
Why any?
I think this checks for empty list. We would save  values that are set to 0. Any if it is not set, we ignore it.